lib/routes/jinritemai/docs.ts

```diff
@@ -0,0 +1,76 @@
+import { Route } from '@/types';
+import cache from '@/utils/cache';
+import got from '@/utils/got';
+import dayjs from 'dayjs';
+import timezone from '@/utils/timezone';
+
+const typeMap = {
+    '5': '全部公告',
+    '19': '产品发布',
+    '21': '规则变更',
+    '20': '维护公告',
+    '22': '其他公告',
+};
+
+/**
+ *
+ * @param ctx {import('koa').Context}
+ */
+export const route: Route = {
+    path: '/docs/:dirId?',
+    categories: ['programming'],
+    example: '/jinritemai/docs/19',
+    parameters: { dirId: '公告分类, 可在页面URL获取 默认为全部' },
+    features: {
+        requireConfig: false,
+        requirePuppeteer: false,
+        antiCrawler: false,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    name: '平台公告',
+    maintainers: ['zhijunchai'],
+    handler,
+    description: `| 类型    | type    |
+  | --------- | ---------- |
+  | 全部公告    | 5    |
+  | 产品发布    | 19   |
+  | 规则变更    | 21   |
+  | 维护公告    | 20   |
+  | 其他公告    | 22   |`,
+};
+
+async function handler(ctx) {
+    const dirId = ctx.req.param('dirId') ? ctx.req.param('dirId') : '5';
+    const limit = ctx.req.param('limit') ? Number.parseInt(ctx.req.param('limit')) : 10;
+    const url = `https://op.jinritemai.com/doc/external/open/queryDocArticleList?pageIndex=0&pageSize=${limit}&status=1&dirId=${dirId}&orderType=3`;
+    const response = await got({ method: 'get', url });
+
+    const list = response.data.data.articles.map((item) => ({
+        title: item.title,
+        id: item.id,
+        dirName: item.dirName,
+        link: `https://op.jinritemai.com/docs/notice-docs/${dirId}/${item.id}`,
+        pubDate: timezone(dayjs.unix(item.createTime).format('YYYY-MM-DD HH:mm:ss'), +8),
```

Comment:
`parseDate` can parse a unit timestamp if you provide `parseDate` a [format string](https://day.js.org/docs/en/parse/string-format)